### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by kadewu**
Based on https://github.com/saleor/saleor/pull/11933

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -102,7 +99,9 @@ jobs:
         if: github.event_name == 'push' && github.repository == 'saleor/saleor'
 
       # Run linters and Django related checks
+      # `git config` command is a workaround for https://github.com/actions/runner-images/issues/6775
       - name: Run Linters and Checks
         run: |
           pre-commit run --all
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```
Comment (NyanKiyoshi) [inline .github/workflows/pytest.yml:106]:
```suggestion
          git config --global --add safe.directory "$GITHUB_WORKSPACE"
          pre-commit run --all
```

File: .github/workflows/pytest.yml [hunk 2]
```diff
@@ -54,11 +54,8 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
 
-      # sphinxcontrib-applehelp added due to https://github.com/coddingtonbear/python-measurement/issues/75.
-      # To remove once the issue is resolved.
       - name: Install dependencies
         run: |
-          python -m pip install sphinxcontrib-applehelp==1.0.2
```
Comment (NyanKiyoshi) [inline .github/workflows/pytest.yml:61]: Is it fixed in this branch?

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]

File: .github/workflows/pytest.yml (hunk 2 above)
Comment (kadewu) [reply .github/workflows/pytest.yml:61]: You are right, I wrongly noticed the version of `measurement` lib.

Decided to update already this library, but if shoudln't then I will revert.

**Review by NyanKiyoshi — APPROVED** [review]

**Review by tomaszszymanski129 — APPROVED** [review]

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]
Can you rebase? Dependencies changed on the base branch

**Review by NyanKiyoshi — APPROVED** [review]

**Pull request merged**